prometheus/value.go

```diff
@@ -110,18 +132,20 @@ func MustNewConstMetric(desc *Desc, valueType ValueType, value float64, labelVal
 }
 
 type constMetric struct {
-	desc       *Desc
-	valType    ValueType
-	val        float64
-	labelPairs []*dto.LabelPair
+	desc   *Desc
+	metric *dto.Metric
```

Comment:
reusing some structs for readability as an effect of https://github.com/prometheus/client_golang/pull/929 experiements.